src/OrchardCore.Cms.Web/appsettings.json

```diff
@@ -44,8 +44,10 @@
     //  "CreateContainer": true // Creates the container during app startup if it does not already exist.
     //},
     // Uncomment to configure content localization.
-    //"OrchardCore_ContentLocalization_CulturePicker": {
```

Comment:
This change is wrong.

The current section name is correct. It should match the `FeatureId`. Refer the `Manifest.cs` for more information.

We already spoke about this on your original pr.

Here you are confusing Features with namespaces, and class names. Which are not connected.

Reply:
I think the current code is ok no ?

Reply:
> This change is wrong.

It's working fine Dean

> The current section name is correct. It should match the FeatureId. Refer the Manifest.cs to understand what a feature is.

I will have a look

> I think the current code is ok. No need for this PR

I will double check